controlplane/src/db/models.ts

```diff
@@ -19,3 +19,60 @@ export type NewOrganizationFeature = typeof organizationFeatures.$inferInsert;
 export type NewBillingSubscription = typeof billingSubscriptions.$inferInsert;
 export type NewBillingPlan = typeof billingPlans.$inferInsert;
 export type MemberRole = (typeof memberRoleEnum.enumValues)[number];
+
+export type AuditableType =
+  | 'organization'
```

Comment:
Should the string we reuse be consts? Not a big deal; just a suggestion.

For instance, 'api_key' is used in a few places.

Reply:
As long as we use the type it is ensured. If you mean something different, please make an example.

Reply:
I just mean reusing a string literal as a constant

```
// constants.ts
const example = 'example';

// file.ts
type X = {
 aString: example
 };
 ```
 
 It's not important. I just see some repeated literals.

Reply:
For exact that reason the type exists. There are two ways to in TS to work with "enums". The traditional way to use  enums or string constants. If you go with string constants you need to write in anytime.